src/assets/icons/close.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="43" height="45" fill="none" viewBox="0 0 43 45">
+  <path fill="#8CA775" d="M25.931 35.625a1.875 1.875 0 0 1-1.462-.694l-9.057-11.25a1.875 1.875 0 0 1 0-2.381l9.375-11.25a1.877 1.877 0 1 1 2.888 2.4L19.294 22.5l8.1 10.05a1.874 1.874 0 0 1-1.463 3.075Z"/>
```

Comment:
do we want the icon to have the #8CA775 Color upfront?

Reply:
yep, is specified in mockups (both light and dark mode are the same)